### PR TITLE
 中英文翻译

### PR DESCRIPTION
**Pull request by h4ck3rm1k3**

### CODE REVIEW

**Comment by lowstz** [comment]
ping @magicshui

**Comment by h4ck3rm1k3** [comment]
hmm, no comment?

**Comment by magicshui** [comment]
@h4ck3rm1k3  sorry, didn't save comment...wait i rewrite it

File: resource_dnspod_record.go [hunk 1]
```diff
@@ -52,10 +52,10 @@ func resourceDnspodRecord() *schema.Resource {
 			"line": &schema.Schema{
 				Type:     schema.TypeString,
 				Optional: true,
-				Default:  "默认",
+				Default:  "Default",
 				ValidateFunc: func(v interface{}, k string) (ws []string, es []error) {
 					value := v.(string)
-					opts := []string{"默认", "国内", "国外", "电信", "联通", "教育网", "移动", "百度", "谷歌", "搜搜", "有道", "必应", "搜狗", "奇虎", "搜索引擎"}
+					opts := []string{"Google", "Yahoo"}
```
Comment (magicshui) [inline resource_dnspod_record.go]: this is the Parameter content,can't be translated

File: resource_dnspod_record.go [hunk 2]
```diff
@@ -52,10 +52,10 @@ func resourceDnspodRecord() *schema.Resource {
 			"line": &schema.Schema{
 				Type:     schema.TypeString,
 				Optional: true,
-				Default:  "默认",
+				Default:  "Default",
```
Comment (magicshui) [inline resource_dnspod_record.go]: can't be translated, otherwise dnspod api will report an error

**Comment by h4ck3rm1k3** [comment]
oh, ok.

**Pull request closed without merging**